src/main/kotlin/racingcar/Driver.kt

```diff
@@ -0,0 +1,31 @@
+package racingcar
+
+class Driver {
+    val reader = Reader()
+
+    fun getValidDriver(): List<String> {
+        return checkCarNameException(inputCarName())
+    }
+
+    private fun inputCarName(): List<String> {
+        return reader.inputNum().split(",")
+    }
```

Comment:
`inputNum`을 통해 입력값들을 받아오는 것 같은데, 함수명을 `userInput`과 같이 정의하고 변수에 할당한 뒤 결과를 내는게 가독성이 좋을 것 같아요!
```suggestion
    private fun inputCarName(): List<String> {
        val carNames = reader.userInput().split(",")
        return carNames 
    }
```